main/acle.md

```diff
@@ -2045,6 +2057,56 @@ support for the SVE2 SM4 (FEAT_SVE_SM4) instructions and if the associated
 ACLE intrinsics are available. This implies that `__ARM_FEATURE_SM4` and
 `__ARM_FEATURE_SVE2` are both nonzero.
 
+### Lookup table extensions
```

Comment:
Looks like this section should be part of a separate PR.

Reply:
Ack, it for/in https://github.com/ARM-software/acle/pull/324